lib/wickedbase.pm

```diff
@@ -363,6 +363,26 @@ sub create_tunnel_with_commands {
     assert_script_run("ip addr");
 }
 
+sub unique_macaddr {
```

Comment:
there are several issues with this function : 
1. I am not sure that such fully random approach will not bring corner cases due to IPv6 specification . AFAIK IPv6 have some IP ranges  which have different purpose and will behave differently. So after all test which using this generator may produce different results based on output of this function. So we probably need not fully blind random here but something more intelligent taking IPv6 specification into account 
2. I assume there should be some perl/ C libraries which doing that smart 
3. artificial limitation which not allow running test on workers with ID greater than X is *NO GO* . Keep in mind that wicked tests are MM and MM tests due to implementation of tun/tap logic getting workers with ID + 64 . But I assume this one will be auto-resolved if we will process first two